.circleci/config.yml

```diff
@@ -82,11 +82,26 @@ jobs:
             - src
 
   checks:
-    executor: php_node
+    executor: php
     steps:
       - attach_workspace:
           at: /tmp
+      - checkout
+      - run:
+          name: Install Composer
+          command: |
+            mkdir -p ~/project/bin
+            cd ~/project/bin
+            echo 'export PATH=$HOME/project/bin:$PATH' >> $BASH_ENV
+            source $BASH_ENV
+            php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+            EXPECTED_SIGNATURE=$(curl -s https://composer.github.io/installer.sig)
+            ACTUAL_SIGNATURE=$(php -r "echo hash_file('sha384', 'composer-setup.php');")
+            [[ "$EXPECTED_SIGNATURE" == "$ACTUAL_SIGNATURE" ]] && php composer-setup.php --install-dir=/root/project/bin --filename=composer || exit 1
+            rm composer-setup.php
+            composer config -g github-protocols https && composer config -g repo.packagist composer https://packagist.org
```

Comment:
We have to manually install composer because I couldn't find a pre-made image that has the right version of `curl`: `8.4.0`.